assets/latex/jlcode.sty

```diff
@@ -0,0 +1,2247 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%
+%%% jlcode.sty
+%%% Copyright 2018 Willi Gerbig
+%%
+%% This work may be distributed and/or modified under the
+%% conditions of the LaTeX Project Public License, either
+%% version 1.3 of this license or (at your option) any
+%% later version.
+%% The latest version of this license is in
+%% http://www.latex-project.org/lppl.txt
+%% and version 1.3 or later is part of all distributions of
+%% LaTeX version 2005/12/01 or later.
+%%
+%% This work has the LPPL maintenance status 'maintained'.
+%%
+%% The Current Maintainer of this work is Willi Gerbig.
+%%
+%% This work consists of the files jlcode.sty, createkwlists.jl,
+%% createucclist.jl, testfile.jl, testfile2.jl and manual.pdf.
+%%
+%%
+%%
+% Keywords, literals, built-ins, macros and functions were created by
+% createkwlists.jl, which was inspired by the following script:
+% https://github.com/isagalaev/highlight.js/blob/master/src/languages/julia.js
+% Colors for the themes come from:
+% https://docs.julialang.org/en/v1/assets/themes/documenter-light.css
+% https://github.com/dietercastel/jlcode/blob/darktheme/jlcode.sty
+% special unicode characters come from:
+% https://docs.julialang.org/en/v1/manual/unicode-input/
+% The JuliaMono font approach was inspired by
+% https://github.com/mossr/julia-mono-listings
+% and the JuliaMono font files were downloaded from:
+% https://github.com/mossr/julia-mono-listings
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% defining the jlcode package
+\def\fileversion{6.1}
+\def\filedate{2022/05/03}
```

Comment:
at the top of the file the copyright is only listed as 2018, here it says 2022...?!